CHANGELOG.md

```diff
@@ -12,6 +12,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+
+## [v0.2.4] - 2020-06-10
+
+### Added
+
+
```

Comment:
Empty on purpose?

Reply:
yeah we haven't added anything new afaik, maintenance only.

Reply:
Not a big fan of an empty section but could live with that.